extensionsGallery-insider.json

```diff
@@ -37,7 +37,7 @@
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.License",
-									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt"
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE"
```

Comment:
Undo this, our repo still has the .txt extension (you need to update the one down below in the section for your extension)